src/main/java/org/motechproject/newebodac/dto/SiteDto.java

```diff
@@ -10,6 +10,8 @@
 public class SiteDto {
 
   @Uuid
+  private String id;
```

Comment:
this should be changed, we should use UUID in all of the DTOs, it's serializable, so it will work fine